macros/staging/get_video_columns.sql

```diff
@@ -39,7 +39,7 @@
     {"name": "status_license", "datatype": dbt.type_string()},
     {"name": "status_made_for_kids", "datatype": "boolean"},
     {"name": "status_public_stats_viewable", "datatype": "boolean"},
-    {"name": "status_publish_at", "datatype": "datetime"},
+    {"name": "status_publish_at", "datatype": dbt.type_timestamp()},
```

Comment:
It looks like we have this field included in the seed data; however, we don't actually have any records. Let's use this as an opportunity to ensure this code change fixes the issue and doesn't cause any issues in the future. 

To do so, let's remove the `status_publish_at` field from the [seed data](https://github.com/fivetran/dbt_youtube_analytics/blob/main/integration_tests/seeds/youtube_video_data.csv). This way the get_video_columns() macro will be triggered to create the null record and cast it as a timestamp. Previously, this would cast as datetime and would cause the failure for databricks sql warehouse. It shouldn't throw any failures following these updates.